src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -464,161 +462,142 @@ const addressUISchema = (addressName, title) => {
-      type: {
-        type: 'string',
-        'enum': ['MILITARY', 'DOMESTIC', 'INTERNATIONAL']
-      },
-      country: {
-        type: 'string',
-        'enum': countries
-      },
-      state: {
-        type: 'string',
-        'enum': states
-      },
-      addressLine1: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      addressLine2: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      addressLine3: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
-      },
-      city: {
-        type: 'string',
-        maxLength: 35,
-        pattern: "([a-zA-Z0-9-'.#]([a-zA-Z0-9-'.# ])?)+$"
-      },
-      zipCode: {
-        type: 'string'
+const addressSchema = {
+  type: 'object',
+  required: ['country', 'addressLine1'],
+  properties: {
+    type: {
+      type: 'string',
+      'enum': ['MILITARY', 'DOMESTIC', 'INTERNATIONAL']
+    },
+    country: {
+      type: 'string',
+      'enum': countries
+    },
+    state: {
+      type: 'string',
+      'enum': states
+    },
+    addressLine1: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    addressLine2: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    addressLine3: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.,,&#]([a-zA-Z0-9-'.,,&# ])?)+$"
+    },
+    city: {
+      type: 'string',
+      maxLength: 35,
+      pattern: "([a-zA-Z0-9-'.#]([a-zA-Z0-9-'.# ])?)+$"
+    },
+    zipCode: {
+      type: 'string'
+    },
+    militaryPostOfficeTypeCode: {
+      type: 'string',
+      'enum': ['APO', 'DPO', 'FPO']
+    },
+    militaryStateCode: {
+      type: 'string',
+      'enum': ['AA', 'AE', 'AP']
+    }
+  }
+};
+
+export const uiSchema = {
+  veteran: {
+    'ui:title': 'Contact information',
+    'ui:field': ReviewCardField,
+    'ui:options': {
+      viewComponent: PrimaryAddressViewField
+    },
+    mailingAddress: addressUISchema('mailingAddress'),
+    primaryPhone: {
+      'ui:title': 'Primary telephone number',
+      'ui:widget': SSNWidget, // TODO: determine whether to rename widget
+      'ui:validations': [validatePhone],
+      'ui:errorMessages': {
+        pattern: 'Phone numbers must be at least 10 digits (dashes allowed)'
       },
-      militaryPostOfficeTypeCode: {
-        type: 'string',
-        'enum': ['APO', 'DPO', 'FPO']
+      'ui:options': {
+        widgetClassNames: 'va-input-medium-large'
+      }
+    },
+    secondaryPhone: {
+      'ui:title': 'Secondary telephone number',
+      'ui:widget': SSNWidget,
+      'ui:validations': [validatePhone],
+      'ui:errorMessages': {
+        pattern: 'Phone numbers must be at least 10 digits (dashes allowed)'
```

Comment:
Should this be 7 instead of 10? Or "Either 7 or 10 digits (dashes allowed)" perhaps a question for @peggygannon ?

Reply:
It actually should be `exactly 10 digits` because that's what EVSS expects. 3 for the `areaCode`, 7 for the `phoneNumber`. Should also check that the validation does actually accept dashes...I think we don't right now, so that's misleading...

At any rate, I don't know that it makes sense to change it in this particular PR because I only modified the whitespace here, not the actual content. What would be the best way to track this work? Should we have a ticket to go through all the validation and make sure the error messages are appropriate?